packages/parser/package.json

```
@@ -0,0 +1,46 @@
+{
+  "name": "mermaid-parser",
+  "version": "0.2.0",
+  "description": "MermaidJS parser",
+  "author": "Yokozuna59",
```

```suggestion
  "contributors": [
    "Yokozuna59",
    "Sidharth Vinod (https://sidharth.dev)"
  ],
```

Would you mind sharing credit :) ?

---

Sure :)